index.html

```diff
@@ -12920,6 +12920,42 @@ <h2>A project for first-time contributions</h2>
                 </div>
             </div>
             <!-- ________ CHINONSO's card END ________  -->
+	
+	<!-- ________ Contributor's card START ________  -->
```

Comment:
You're just one step away. Just a nitpick - could you please fix this indentation to be in line with the rest of the code?

Reply:
I did the changes as you asked. Can you please check. Thank you so much!